README.md

```diff
@@ -57,3 +57,18 @@ Ansible: There are Ansibles files in the Ansible folder and you can run the comm
 ansible-playbook my-ansible-playbook.yml -i inventory.txt
 
 ```
+## Docker Module 5
+```
+Command in the terminal to build the image
+docker build --target development --tag todo-app:dev .    
+
+(Development Image)
+ docker run --env-file ./.env -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
+
+(Production Image )
+docker build --target production --tag todo-app:prod . 
+docker run --env-file ./.env -p 5000:5000 todo-app:prod
+
+misc:CMD ["poetry", "run", "flask","run","--host=0.0.0.0"] (to run on flask only )
```

Comment:
I _think_ this is probably unnecessary to document since it's now your development CMD